app/src/components/TermsOfUse.jsx

```diff
@@ -16,405 +17,407 @@ class TermsOfUse extends Component {
+                      CONTAINED IN THESE TERMS APPLY TO YOU TO THE FULLEST EXTENT SUCH LIMITATIONS
+                      OR EXCLUSIONS ARE PERMITTED UNDER THE LAWS OF THE JURISDICTION IN WHICH YOU ARE LOCATED.
+                      </p>
+                    </li>
+                  </ol>
+                </li>
+                <li>
+                  <h2>Digital Millennium Copyright Act Compliance</h2>
+                  <p>
+                    If You are a copyright owner or an agent thereof, and believe that any Site
+                    content infringes upon Your copyrights, You may submit a notification
+                    pursuant to the Digital Millennium Copyright Act (“DMCA”) by providing
+                    our Copyright Agent (as defined below) with the following information
+                    in writing (see 17 U.S.C § 512(c)(3) for further detail):
+                  </p>
+                  <ol className={TextPages['second-level-list']}>
+                    <li>
+                      <p>
+                      A physical or electronic signature of a person authorized
+                      to act on behalf of the owner of an exclusive right that is allegedly infringed;
+                      </p>
+                    </li>
+                    <li>
+                      <p>
+                        Identification of the exact location on the Site of copyrighted work
+                        claimed to have been infringed, or, if multiple copyrighted works on the
+                        Site are covered by a single notification, a representative list of such
+                        works as located on the Site;
+                      </p>
+                    </li>
+                    <li>
+                      <p>The address, telephone number and electronic mail address of the complaining party</p>
+                    </li>
+                    <li>
+                      <p>
+                        A statement that the complaining party has
+                        a good faith belief that use of the material in the
+                        manner complained of is not authorized by the copyright owner, its agent or the law
+                      </p>
+                    </li>
+                    <li>
+                      <p>
+                        Your electronic or physical signature (as appropriate).
+                      </p>
+                    </li>
+                  </ol>
+                  <p>
+                    The designated Copyright Agent to receive notifications of claimed infringement is:
+                    SkyTruth Tel: <a className={AppStyles['-underline']} href="tel:1 (304) 885-4581">
+                    +1 (304) 885-4581</a> Email: <a
                       className={AppStyles['-underline']}
-                      href="http://earthengine.google.org/terms" target="_blank"
+                      href="dmcaagent@skytruth.org"
```

Comment:
For this we use `mailto:email` in the `href`. This way, the default mail app in the system will open and create a new email with the email address attached.